src/util.ts

```diff
@@ -54,5 +54,6 @@ export const getUserAgent = (): string => {
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const packageVersion = require('../package.json').version;
   const platformName = os.platform();
-  return `sigstore-js/${packageVersion} (${platformName})`;
+  const nodeVersion = process.version;
+  return `sigstore-js/${packageVersion} (Node ${nodeVersion}) (${platformName})`;
```

Comment:
Since we're in here making changes, let's add the `process.arch` value as well. So we'll end-up with something like:
```
sigstore-js/0.0.0 (Node v16.15.0) (darwin/arm64)
```